From dea0b9c8e8767d49af81eb975c818bcf9d2e674a Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Wed, 12 Feb 2025 09:03:42 +0100
Subject: [PATCH] shim: Define RTLD_DEEPBIND if not already

RTLD_DEEPBIND is a glibc quirc and is not defined on other libc like
musl libc
---
 src/gl/shim.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/gl/shim.c b/src/gl/shim.c
index 5ac335ed4d..98b61e9d56 100644
--- a/src/gl/shim.c
+++ b/src/gl/shim.c
@@ -9,6 +9,10 @@
 #include <stdbool.h>
 #include <stdlib.h>
 
+#ifndef RTLD_DEEPBIND
+#define RTLD_DEEPBIND 0
+#endif
+
 static void* handle = NULL;
 static bool mangoHudLoaded = false;
 
